wfl/autoparallelize/base.py

```diff
@@ -86,14 +86,14 @@ def iloop(func, *args, def_npool=None, def_chunksize=1, iterable_arg=0, def_skip
     remote_info = kwargs.pop('remote_info', def_remote_info)
     label = kwargs.pop('label', def_label)
 
-    return iterable_loop(npool, chunksize, inputs, outputs, func, iterable_arg, skip_failed,
+    return autoparallelize(npool, chunksize, inputs, outputs, func, iterable_arg, skip_failed,
```

Comment:
Did we talk about looking for clearer names for `chunksize` and `job_chunksize`? 

not sure if any of these are any better for `chunksize`: `serial_chunk_size`/`serial_chunksize`/`num_serial`/`num_serial_items`/`num_serial_ops`/`num_operations_in_serial`

instead of `job_chunksize` maybe `items_per_job`?

Reply:
Yes, we did, but I think that could also be a different PR. We need to finish the workflow presentation/discussion,